docs/node/failures.md

```diff
@@ -0,0 +1,68 @@
+---
+id: failures
+title: Failures in Node
+sidebar_label: Failures
+---
```

Comment:
Just a suggestion. the other thing we can name it is "Handling Failures"

```suggestion
---
id: failure-classes
title: Failure Classes in Node
sidebar_label: Failure Classes
---
```

Reply:
I like the handling failures idea, I'll change.